### ISSUE TITLE
Hepatic xanthomas (SimulConsult code elthat_101120200536)

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9624]
**Preferred term label:**
Hepatic xanthomas

**Synonyms**
Liver xanthomas

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cerebrotendinous xanthomatosis
Cholesteryl ester storage disease
Wolman disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
Hepatic xanthomatosis
HP_0034621
Hepatic xanthomas - exact synonym
